.circleci/config.yml

```diff
@@ -100,7 +100,7 @@ jobs:
                   --message "Docs build of $CIRCLE_SHA1" \
                   --force
             else
-              echo "Not on the master branch; skipping deployment"
+              echo "Not on the main branch; skipping deployment"
```

Comment:
Do the references in the opening conditional also need to be updated: `"master"` -> `"main"` too?

Reply:
Good catch, fixed.